libs/cli/langchain_cli/integration_template/integration_template/vectorstores.py

```diff
@@ -26,15 +26,106 @@
 
 
 class __ModuleName__VectorStore(VectorStore):
-    """__ModuleName__ vector store.
+    # TODO: Replace all TODOs in docstring.
+    """__ModuleName__ vector store integration.
 
-    Example:
+    # TODO: Replace with relevant packages, env vars.
+    Setup:
+        Install ``__package_name__`` and set environment variable ``__MODULE_NAME___API_KEY``.
+
+        .. code-block:: bash
+
+            pip install -U __package_name__
+            export __MODULE_NAME___API_KEY="your-api-key"
+
+    # TODO: Populate with relevant params.
+    Key init args — indexing params:
+        collection_name: str
+            Name of the collection.
+        embedding_function: Embeddings
+            Embedding function to use.
+
+    # TODO: Populate with relevant params.
+    Key init args — client params:
+        client: Optional[Client]
+            Client to use.
+        connection_args: Optional[dict]
+            Connection arguments.
+
+    # TODO: Replace with relevant init params.
+    Instantiate:
         .. code-block:: python
 
             from __module_name__.vectorstores import __ModuleName__VectorStore
 
-            vectorstore = __ModuleName__VectorStore()
-    """
+            vector_store = __ModuleName__VectorStore(
+                collection_name="foo",
+                embedding_function=OpenAIEmbeddings(),
+                connection_args={"uri": "./foo.db"},
+                # other params...
+            )
+
+    # TODO: Populate with relevant variables.
+    Add Documents:
+        .. code-block:: python
+
+            from langchain_core.documents import Document
+
+            document = Document(page_content="foo", metadata={"baz": "bar"})
+            vector_store.add_documents([document],ids=["1"])
+
+    # TODO: Populate with relevant variables.
+    Update Documents:
+        .. code-block:: python
+
+            updated_document = Document(
+                page_content="qux",
+                metadata={"bar": "baz"}
+            )
+
+            vector_store.update_documents(document_id="1",document=updated_document)
+
+    Delete Documents:
+        .. code-block:: python
+
+            vector_store.delete(ids=["1"])
+
+    # TODO: Fill out with relevant variables and example output.
+    Search:
+        .. code-block:: python
+
+            results = vector_store.similarity_search(query="thud",k=1,filter={"source":"https://example.com"})
```

Comment:
not all vecstores support filtering, and even for ones that do there's not a standard filter syntax. so would drop that part